Dockerfile

```diff
@@ -1,9 +1,15 @@
-FROM quay.io/openshift/origin-must-gather:4.4.0 as builder
+FROM golang:1.14.4 as gobuilder
+
+RUN go get github.com/google/pprof
+
+FROM quay.io/openshift/origin-must-gather:4.5.0 as builder
```

Comment:
Just 4.5 so we get z-stream updates, please.

Reply:
@jmontleon updated